### ISSUE TITLE
[Docs] import get from openapi-v3 or rest ?

### ISSUE THREAD

**Issue by rfreydi** [strongloop/loopback-next#1418]
Currently in the [getting-started](https://github.com/strongloop/loopback-next/blob/master/docs/site/Getting-started.md#adding-your-own-controller) where we define a new controller we have an import for the get decorator:
```
import {get} from '@loopback/rest';
```
But the CLI and the autocomplete of PhpStorm import it from another package:
```
import {get} from '@loopback/openapi-v3';
```

Which one should be used ?

Here is the output of `lb4 -v`:
```
maps@maps-romain:~/personnel_prive/development/loopback$ lb4 -v
@loopback/cli version: 0.13.6

@loopback/* dependencies:
  - @loopback/authentication: ^0.10.9
  - @loopback/boot: ^0.10.9
  - @loopback/build: ^0.6.8
  - @loopback/context: ^0.11.6
  - @loopback/core: ^0.8.8
  - @loopback/metadata: ^0.8.7
  - @loopback/openapi-spec-builder: ^0.7.7
  - @loopback/openapi-v3-types: ^0.7.7
  - @loopback/openapi-v3: ^0.10.9
  - @loopback/repository-json-schema: ^0.9.9
  - @loopback/repository: ^0.11.3
  - @loopback/rest: ^0.11.3
  - @loopback/testlab: ^0.10.7
  - @loopback/docs: ^0.9.6
  - @loopback/example-hello-world: ^0.8.4
  - @loopback/example-log-extension: ^0.10.4
  - @loopback/example-rpc-server: ^0.8.4
  - @loopback/example-todo: ^0.11.4
  - @loopback/dist-util: ^0.3.2
  - @loopback/service-proxy-proxy: ^0.4.0
  - @loopback/service-proxy: ^0.5.9
  - @loopback/http-caching-proxy: ^0.2.3
  - @loopback/http-server: ^0.1.3
```

And here are my current dependencies of the package.json:
```
  "dependencies": {
    "@loopback/context": "^0.11.6",
    "@loopback/boot": "^0.10.9",
    "@loopback/core": "^0.8.8",
    "@loopback/dist-util": "^0.3.2",
    "@loopback/rest": "^0.11.3",
    "@loopback/openapi-v3": "^0.10.9"
  },
  "devDependencies": {
    "@loopback/build": "^0.6.8",
    "@types/node": "^10.1.1",
    "@loopback/testlab": "^0.10.7",
    "@types/mocha": "^5.0.0",
    "mocha": "^5.1.1",
    "source-map-support": "^0.5.5"
  }
```

**Comment by shimks** [comment]
`rest` reexports the openapi-v3 decorators; this was done so that an application developer could export all `rest` related artifacts from a single source. We thought that there being two sources of the decorators could be confusing for the users, so #1117 has been created to track better documentation of the issue.

TLDR; import from `rest` if you're an app dev, import from `openapi-v3` if you're building an openapi extension
